packages/daemon/test/test-endo.js

```diff
@@ -1348,9 +1348,8 @@ test('read remote value', async t => {
   const hostA = await prepareHostWithTestNetwork(t);
   const hostB = await prepareHostWithTestNetwork(t);
 
-  // introduce nodes to each other
+  // Introduce A to B (such that B becomes connected to A consequently.)
   await E(hostA).addPeerInfo(await E(hostB).getPeerInfo());
-  await E(hostB).addPeerInfo(await E(hostA).getPeerInfo());
```

Comment:
This explicitly aggravates a crossed-hello and results in data-lock. I would like to punt the general solution in hopes that daemons will rarely restart at the same time. I am aware that this is foolish in the long term, but crossed hellos are hard and time is running short for our proof-of-concept demo.